src/main/java/gov/nasa/jpl/mbee/mdk/mms/MMSUtils.java

```diff
@@ -612,8 +619,8 @@ private static URIBuilder getServiceUri(Project project, String baseUrl) {
             e.printStackTrace();
             return null;
         }
-
-        uri.setPath("/alfresco/service");
+        String currentPath = Optional.ofNullable(uri.getPath()).orElse("");
+        uri.setPath(currentPath + (currentPath.endsWith("/") ? "" : "/") + "alfresco/service");
```

Comment:
Unless I'm making a logic error. this would not handle properly the case where the user already had "alfresco/service" in the path. We should probably strip or not modify if that is the case.

Reply:
Good point. Addressed that case.